Make.inc

```diff
@@ -606,6 +606,17 @@ else
 ISX86:=0
 endif
 
+# If we are running on powerpc64, set certain options automatically
+ifneq (,$(findstring powerpc64,$(ARCH)))
+JCFLAGS += -fsigned-char
+
+override LLVM_VER:=3.8.0
+override OPENBLAS_DYNAMIC_ARCH:=0
+override OPENBLAS_TARGET_ARCH:=POWER8
+override USE_SYSTEM_LIBM:=1
```

Comment:
It does, but Julia pulls in an older version. Could we update the version Julia uses?